nautobot/docs/additional-features/jobs.md

```diff
@@ -533,15 +533,18 @@ When providing input data, it is possible to specify complex values contained in
 
 ### Via the CLI
 
-Jobs that do not require user input can be run from the CLI by invoking the management command:
+Jobs can be run from the CLI by invoking the management command:
 
 ```no-highlight
-nautobot-server runjob [--username <username>] [--commit] <class_path>
+nautobot-server runjob [--username <username>] [--commit] [--data <data>] <class_path>
 ```
 
 !!! note
     [See above](#jobs-and-class_path) for `class_path` definitions.
 
+!!! note
+    The `--data` parameter should be constructed as a JSON serializable string, e.g. `--data='{"string_variable": "somevalue", "integer_variable": 123}'`
```

Comment:
```suggestion
    The `--data` parameter should be a JSON representation of the job's inputs, e.g. `--data='{"string_variable": "somevalue", "integer_variable": 123}'`
```

Reply:
What is a `JSON representation`? I don't feel that provides the user with an actionable response. A quick google search of JSON serializable string brings up relevant Stack Overflow links, whereas JSON representation does not.

Reply:
Will resolve both of these tomorrow.

Reply:
```suggestion
    The `--data` parameter must be a JSON string, e.g. `--data='{"string_variable": "somevalue", "integer_variable": 123}'`
```